src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsFeed.java

```diff
@@ -221,8 +224,9 @@ private String getSaveFilePath(String gtfsFeedUrl, HttpURLConnection connection)
         } else {
             //Extracts file name from header field
             int index = disposition.indexOf("filename=");
+            int filenameLastIndex = disposition.indexOf("\"", index+10);
```

Comment:
If you could add a unit test for loading GTFS data as well that would be great. Travis is Linux based so the test would fail on Travis if the code isn't platform independent.